client/app/startup/registration.jsx

```diff
@@ -3,7 +3,7 @@
  * Add components only if needed in current application, because
  * this bundle will be loaded in production.
  */
-
+import '@babel/polyfill';
```

Comment:
You can move this into the "entry" webpack parameter. e.g.
https://github.com/baohouse/labs.baohouse.net/blob/master/webpack.config.js

Reply:
Ooh you're right, thanks for the callout